test/unit/math/prim/mat/fun/add_diag_test.cpp

```diff
@@ -59,29 +59,29 @@ TEST(MathPrimMat, double_mat_double_rvec_add_diag) {
 }
 
 TEST(MathPrimMat, var_mat_double_vec_add_diag) {
```

Comment:
This is a duplicate of a test above, right?

Reply:
yep good catch